Raygun4UWP/RaygunClient.cs

```diff
@@ -0,0 +1,455 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.IO;
+using System.Linq;
+using System.Threading.Tasks;
+using Windows.Networking.Connectivity;
+using Windows.Security.ExchangeActiveSyncProvisioning;
+using Windows.Storage;
+using Windows.UI.Xaml;
+using Windows.Web.Http;
+using Newtonsoft.Json;
+
+namespace Raygun4UWP
+{
+  public class RaygunClient
+  {
+    private const string OFFLINE_DATA_FOLDER = "Raygun4UWPOfflineCrashReports";
+
+    private bool _handlingRecursiveErrorSending;
+
+    /// <summary>
+    /// Creates a new instance of the <see cref="RaygunClient" /> class.
+    /// </summary>
+    /// <param name="apiKey">The API key.</param>
+    public RaygunClient(string apiKey)
+    {
+      Settings = new RaygunSettings(apiKey);
+
+      BeginSendStoredCrashReports();

```

Comment:
I don't think this should be called in the constructor as it would be called if users decide to new up their own instances of the client. How about moving this to the `Initialize` method? E.g.
```
if (Current == null)
{
  Current = new RaygunClient(apiKey);
  Current.BeginSendStoredCrashReports();
}
```

Reply:
Good idea, though I added it to EnableCrashReporting instead.